components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from 'react';
+import { ArrowLeft, ArrowRight } from 'react-feather';
+
+import {
+  PaginationButtonGroupProps,
+  PaginationButtonProps,
+  PaginationNumberButtonProps,
+  PaginationProps
+} from '@/components/Pagination/Pagination.types';
+import { calculatePagination } from '@/utils/pagination.util';
+
+const PrevButton = (props: PaginationButtonProps) => {
```

Comment:
We can destruct props and use props directly into components. Not like `props.something`